### ISSUE TITLE
iOS version goes above one digit

### ISSUE THREAD

**Issue by Tickthokk** [nadangergeo/RWD-Table-Patterns#44]
```
var iphone_version= parseFloat(String(window.navigator.userAgent.match(/[0-9]_[0-9]/)).split('_')[0]+'.'+String(window.navigator.userAgent.match(/[0-9]_[0-9]/)).split('_')[1]);
// iPhone CPU iPhone OS 8_4 like Mac OS X
return iphone_version;
```

This returns `1` on my iPhone6 test device (which is at OS version 11), which winds up setting `useFixedSolution` to `false`, which causes some animation issues.

Here's the devices user agent string:

`Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1`

I added a custom fix of `[0-9]+` on my end, but I'm sure you'd like to officially fix this.

Thanks again for the plugin!

**Comment by nadangergeo** [comment]
Hi @Tickthokk!

What version of the plugin are you using? In the latest version, getIOSVersion() returns 11 with that user agent. Besides, I could not find the code you referred, which makes me think it might be an old version.

No worries, I'm glad you like it!

**Comment by Tickthokk** [comment]
That makes sense.  I'm using 5.2.  But I had to investigate because I've only been using this plugin for a month, so it's odd that I'm a subversion+ off from the latest 5.3.2.

I downloaded the "latest" from your Github Releases page (https://github.com/nadangergeo/RWD-Table-Patterns/releases).  In hindsight I should have just grabbed master 😛 

I did that because the download link on the demo page 404's (https://codeload.github.com/nadangergeo/RWD-Table-Patterns/zip/v5.3.2)

I'll let you close this ticket in case you wanted to keep it open to resolve the releases and 404, but otherwise you did resolve the core issue I was submitting for.
